infrastructure/index.ts

```diff
@@ -583,7 +577,7 @@ const distributionArgs: aws.cloudfront.DistributionArgs = {
             pathPattern: '/ai/*',
             originRequestPolicyId: allViewerExceptHostHeaderId,
             cachePolicyId: cachingDisabledId,
-            lambdaFunctionAssociations: [],
+            lambdaFunctionAssociations: config.doAIAnswersRewrites ? [getAIAnswersRewriteAssociation()] : [],
```

Comment:
My main concern is whether there will be any latency/streaming impact - if this is deployed to staging first can we verify it there?

Reply:
Makes sense yeah. It's currently deployed to my dev stack here if you want to have a look: 

https://www-christian.pulumi-dev.io/ai